src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/postsubmit/EndApplicationConfirmation.java

```diff
@@ -34,7 +34,7 @@ public PostSubmitCallbackResponse handle(
         postSubmitResponse.setConfirmationHeader("# You have ended the application");
         postSubmitResponse.setConfirmationBody(
             "#### What happens next\n\n"
-                + "A notification has been sent to all parties. "
+                + "A notification has been sent to all sendDirectionList. "
```

Comment:
The word "parties" got accidentally swapped for "sendDirectionList"

Reply:
I have changed sendDirectionList back to parties